### DISCUSSION TITLE
Questions about install to usage

### DISCUSSION THREAD

**Discussion by datatalking** [hydrosquall/tiingo-python#672] [Q&A]
I'm a visual learner so the pages and pages of text just become word salad after a while I need pictures. I'm willing to update the documentation but I need a clearer understanding of what Tiingo is.

So the [api.Tiingo.com ](https://api.tiingo.com) website shows a GUI 
![Screen Shot 2021-08-25 at 7 35 21 PM](https://user-images.githubusercontent.com/11813209/130891424-8f582d57-76f9-4844-9db4-774995b875ed.png)

The actual GUI, is www.Tiingo.com, but where is that getting its data feed from?

So there is no GUI of the python we are working on but does this feed Tiingo and how do you use this data in practice. Are you integrating this into a backtrader pipeline that feeds into interactivebroker, or TDAmeritrade or?

I see Tiingo has access to intraday data which I need since I lost my MorningStar feed, but I'm not able to follow

**Comment by hydrosquall** [comment]
Hi @datatalking , thanks for reaching out. I'm not sure I follow all the questions, but hopefully this helps.

> how do you use this data in practice. 

Would these example notebooks help? https://github.com/hydrosquall/tiingo-python/tree/master/examples

> Are you integrating this into a backtrader pipeline that feeds into interactivebroker, or TDAmeritrade or?

We don't connect to these trading systems directly. You could write a program that used the API clients for those systems, and use the Tiingo API to make decisions about which commands to send to those systems.
